docs/user-guides/tutorials/start.md

```diff
@@ -0,0 +1,73 @@
+# kindly start
+
+## Fist steps
+
+For a beginner in programming with no background in the area, a quick immersion in the package begins with a direct and
+[straightforward introduction](https://medium.com/@katabathina44313/a-gentle-introduction-to-the-world-of-python-programming-eae2b4bc6fc6)
+to Python and how to install it.
+
+After installing Python, the user should install an IDE. An IDE (Integrated Development Environment) is a 
+specialized text editor for coding. We recommend using PyCharm. To get a quick introduction to this IDE, 
+we suggest reading the [Quick start guide](https://www.jetbrains.com/help/pycharm/quick-start-guide.html).
+
+
+## Using flowchem
+
+Diving into an example is the best way to understand the package's features. Let's say you are a chemist who wants to 
+synthesize a molecule C, and to do so, two reagents, A and B, need to come into contact in a reactor. Take a look at 
+the figure below that schematizes the platform. 
+
+![](img.png)
+
+You have three devices on this platform: two pumps and a sensor. To conduct the experiment, you need to activate the 
+pump at a specific flow rate for a set duration, and then activate the sensor to analyze the product. 
+
+And how do you do this? Well, the best approach is to have a drive to access the functionality of these devices in one 
+platform. This task can be accomplished through flowchem.
+
+Using flowchem, it is possible to access the devices' features asynchronously and on a server simply and directly. But 
+what do these terms mean in practice? 
+
+1 - Asynchronous operation means that each device has its own thread, enabling them to run independently of one another.
+This helps prevent issues with one device from affecting the entire platform.
+
+2 - Running on a server provides the platform with great versatility. For instance, the devices can be accessed by 
+another computer on the same network.
+
+In the previous example, to configure or inform Flowchem about the devices you want to access, you just need to write a 
```

Comment:
get consistent in capitalisation of package name

Reply:
just search all for case sensitive Flowchem, FlowChem and change all to flowchem

Reply:
So, if it is placed at the beginning of a phrase, it should be Flowchem.
I verify it!